cv.tex

```diff
@@ -44,92 +50,85 @@
 % Education
 %------------------------------------------------
 
-\section{Education} 
-
-\subsection{Cornell University}
-
-\descript{MEng in Computer Science}
-\location{Expected Dec 2014 | Ithaca, NY \\ Cum. GPA: N/A}
+\section{Education}
+\subsection{\href{https://www.ucf.edu/}{UCF}}
```

Comment:
(JC):
Spell out UCF, assuming it’d fit on the one line.

Reply:
ty